.github/workflows/build-and-test.yml

```diff
@@ -82,9 +87,34 @@ jobs:
                   python -m pip install --upgrade pip nose setuptools
                   python -m pip install -r requirements.txt -r requirements-dev.txt
                   python setup.py install
+                  echo ::set-env name=PYTHON::$(where python)
```

Comment:
Trying to define a global variable with the os-specific python. I wonder if there is a better way.

Reply:
Does this set the `$PYTHON` environment variable?  I haven't seen this notation before ... neat if it does!